py/mpstate.h

```diff
@@ -167,6 +167,11 @@ typedef struct _mp_state_vm_t {
     mp_obj_dict_t *mp_module_builtins_override_dict;
     #endif
 
+    #if MICROPY_PERSISTENT_CODE_TRACK_RELOC_CODE
+    // An mp_obj_list_t that tracks relocated native code to prevent the GC reclaim them.
```

Comment:
to prevent the GC *from reclaiming* them.

Reply:
thanks!  fixed in latest